src/Component/PharUpdate/Console/Command.php

```diff
@@ -91,7 +91,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
         $output->writeln('Looking for updates...');
 
-        /** @var $pharUpdate Helper */
+        /** @var Helper */
```

Comment:
in inline var-phpdocs its invalid to note a variable name.
per spec inline var does define the type of the next var after the doc.
(phpstorm has some non-standard behaviour here, which allows a var name)